app/src/app/[lng]/[inventory]/data/[step]/page.tsx

```diff
@@ -775,15 +785,17 @@ export default function AddDataSteps() {
                         boxSize={6}
                         color="interactive.control"
                       />
-                      {formatPercentage(currentStep.reasonNEProgress)}% <Trans t={t}>not-estimated</Trans>
+                      {formatPercentage(currentStep.reasonNEProgress)}%{" "}
+                      <Trans t={t}>not-estimated-progress</Trans>
                     </Badge>
                     <Badge w="auto">
                       <Box
-                        boxSize={6}
+                        boxSize={3}
                         borderRadius="full"
                         backgroundImage="repeating-linear-gradient(45deg, #D1D5DB, #D1D5DB 2px, transparent 2px, transparent 4px)"
                       />
-                      {formatPercentage(currentStep.reasonNOProgress)}% <Trans t={t}>not-occurring</Trans>
+                      {formatPercentage(currentStep.reasonNOProgress)}%{" "}
+                      <Trans t={t}>not-occurring</Trans>
```

Comment:
Why was the first translation key changed to include the `-progress` suffix but the other one wasn't?
Also make sure to update the respective translation files in other languages to remove the old key (if necessary), as I already merged the translation PR that added them.